plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -791,11 +789,8 @@ private UpdateSite.Data scanBom(HashMap<String, String> pluginGroupIds, String p
     private List<File> getBomEntries() throws IOException, XmlPullParserException, PomExecutionException {
         File fullDepPom = new MavenBom(config.getBom()).writeFullDepPom(config.workDirectory);
 
-    	MavenRunner.Config mconfig = new MavenRunner.Config();
-    	mconfig.userSettingsFile = config.getM2SettingsFile();
+        MavenRunner.Config mconfig = new MavenRunner.Config(config);
     	System.out.println(mconfig.userSettingsFile);
-    	// TODO REMOVE
```

Comment:
Comment originally introduced in #202 by @imonteroperez with no further explanation. It is not clear to me whether this really proposed removing all Maven properties from this `mvn` run, and if so, why we would want that. Possibly this was copy-pasted from other places where the comment apparently refers to `failIfNoTests=false`?